Puzzlepart/prosjektportalen365
# Add predefined help content and support help content on portfolio level

### Your checklist for this pull request


### Description

Updated template for help content list to include datarows, so the list comes with some pre populated elements.

### How to test

1. Apply the list template, or install a new PP365 instance. 
2. Go to configuration site and see the list "Hjelpeinnhold".
3. Go to any of the navigation elements in portfolio or a project.


### Relevant issues (if applicable)
closes #799 

💔Thank you!
